session/bootstrap.go

```diff
@@ -1790,6 +1792,13 @@ func upgradeToVer87(s Session, ver int64) {
 	doReentrantDDL(s, CreateAnalyzeJobs)
 }
 
+func upgradeToVer88(s Session, ver int64) {
+	if ver >= version88 {
+		return
+	}
+	doReentrantDDL(s, "ALTER TABLE `mysql.user` CHANGE `Repl_slave_priv` `Repl_slave_priv` ENUM('N','Y') NOT NULL DEFAULT 'N' AFTER `Create_Tablespace_Priv`")
+}
+
```

Comment:
Getting an error in CI:
```
[](https://ci.pingcap.net/blue/organizations/jenkins/tidb_ghpr_check/detail/tidb_ghpr_check/59947/pipeline#step-122-log-355)[2022-04-18T11:57:54.822Z] session/bootstrap.go:1795:6: `upgradeToVer88` is unused (deadcode)
[2022-04-18T11:57:54.822Z] func upgradeToVer88(s Session, ver int64) {
```

As well as incrementing the currentBootstrapVersion, you'll need to add this to the list of bootstrap tasks.

Reply:
done.

Reply:
done.